cmd/channel/email/main.go

```diff
@@ -80,7 +83,54 @@ func (ch *Email) SetConfig(jsonStr json.RawMessage) error {
 }
 
 func (ch *Email) GetInfo() *plugin.Info {
-	return &plugin.Info{Name: "Email"}
+	minValue := types.Int{NullInt64: sql.NullInt64{Int64: 0}}
+	minValue.Valid = true
+
+	maxValue := types.Int{NullInt64: sql.NullInt64{Int64: 65535}}
+	maxValue.Valid = true
```

Comment:
You can just set `Valid` as well within the struct initializer (and then also move it into the initialization below so you don't need the extra variables).

Reply:
Note: `types.Int` is buggy and ignores the `omitempty` flag.

Reply:
Not sure if that's a bug of `types.Int` or just a limitation of `encoding/json`. Anyways, looks like it just writes `null` to the JSON then. Would that be a problem for the web module?

Reply:
No, Web can handle it.